### DISCUSSION TITLE
Doubt in Resampling

### DISCUSSION THREAD

**Discussion by ansenpjames** [cafawo/FinancialDataAnalytics#4] [Q&A] [answered]
A small doubt in resampling Task 
(Question - "As you can see, we are using daily data so far. Re-sample the data to monthly frequency and plot monthly prices for 'BMWG.DE'.")

what would be the values to each month after resampling? is it average, or last values of the each month....

**Comment by cafawo** [comment, marked as answer]
Make a choice and provide brief reasoning as a comment.
